src/main/kotlin/de/livepoll/api/entity/dto/MultipleChoiceItemDtoIn.kt

```diff
@@ -3,8 +3,16 @@ package de.livepoll.api.entity.dto
 data class MultipleChoiceItemDtoIn(
         val pollId: Long,
         val question: String,
-        val position: Int,
         val allowMultipleAnswers: Boolean,
         val allowBlankField: Boolean,
         val answers: List<String>,
-)
\ No newline at end of file
+)
+
+data class MultipleChoiceItemWithPositionDtoIn(
```

Comment:
I would extract this in a seperate file and use inheritance to get all the fields from `MultipleChoiceItemDtoIn`

Reply:
Thanks, I've adopted the inheritance strategy. However, I don't think we should extract this into three separates files. The Dto is already cluttered and this new class is just a variation of the existing Dto In for the poll item, so I guess it's ok to put them into the same file.